owncloud/owncloud.py

```diff
@@ -589,7 +590,7 @@ def __put_file_chunked(self, remote_path, local_source_file, **kwargs):
                 headers=headers
             )
 
-        chunk_count = size / chunk_size
+        chunk_count = math.ceil(size / chunk_size)
```

Comment:
python2: `math.ceil(float(10)/float(3))` => 4.0
python2: `int(math.ceil(float(10)/float(3)))` => 4
python3: `math.ceil(float(10)/float(3))` => 4

that will do